mne/report.py

```diff
@@ -765,7 +765,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             # on some version mayavi.core won't be exposed unless ...
             from mayavi import mlab  # noqa, mlab imported
             import mayavi
-        except ImportError:
+        except:  # on some systems importing Mayavi raises SystemExit (!)
```

Comment:
... and usually you don't want to catch `SystemExit` so you should at least do `except Exception`, but again here we want to catch everything.